Solutions/CohesitySecurity/Playbooks/Cohesity_Delete_Incident_Blobs/azuredeploy.json

```diff
@@ -0,0 +1,228 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "Delete incident blobs",
+        "description": "This playbook deletes the blobs on Azure storage created by an incident that is generated by Cohesity function apps.",
+        "prerequisites": "Deploy the Cohesity configuration and function apps (see details at https://github.com/cohesity/Azure-Sentinel/blob/CohesitySecurity.internal/Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/readme.md)",
+        "postDeployment": [ "Authorize all connections" ],
+        "prerequisitesDeployTemplateFile": "None",
+        "lastUpdateTime": "2023-01-27T10:57:00.000Z",
+        "entities": [ "Malware" ],
+        "tags": [ "SOAR", "Email Notification", "Threat Response" ],
```

Comment:
I don't think that these tags reflect the meaning of the book. Please think of a new tags.

Reply:
Done.